git-hooks/pre-commit

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+golangci-lint run --tests=0 ./...
```

Comment:
We should instruct people how to install golangci-lint:

https://golangci-lint.run/usage/install/#local-installation

Reply:
Would be only for contributors right?

Reply:
correct; only for the contributors